server/interpreter.js

```diff
@@ -4317,26 +4317,28 @@ Interpreter.prototype.installTypes = function() {
       this.defineProperty('length', Descriptor.none.withValue(options.length),
                           owner);
     }
-    if (options.call) {
-      this.call = options.call;
-    }
-    if (options.construct) {
-      this.construct = options.construct;
-    }
-    if (options.name || options.id) {
-      var id = (options.id || options.name);
+    // Register builtin if possible.
+    var id = (options.id || options.name);
```

Comment:
Unneeded parens.

Reply:
Indeed, and they make the intent less clear because they sort of suggest `id` may be being set to a boolean value, which is not at all the case.  Fixed.